README.md

```diff
@@ -45,32 +41,29 @@ in.
 For more complete docs on developing with Puppet, take a look at the
 rest of the [developer documents](https://github.com/puppetlabs/puppet/blob/master/docs/index.md).
 
-Licensing
----------
+## Licensing
 
 See [LICENSE](https://github.com/puppetlabs/puppet/blob/master/LICENSE) file. Puppet is licensed by Puppet, Inc. under the Apache license. Puppet, Inc. can be contacted at: info@puppet.com
 
-Support
--------
+## Support
 
 Please log tickets and issues at our [JIRA tracker](https://tickets.puppetlabs.com).  A [mailing
 list](https://groups.google.com/forum/?fromgroups#!forum/puppet-users) is
-available for asking questions and getting help from others. In addition, there
-is an active #puppet channel on Freenode.
+available for asking questions and getting help from others.
```

Comment:
Should we include a referenced to slack here since we've replaced irc with that?

Reply:
@melissa updated!